android/sdl_android/src/main/java/com/smartdevicelink/managers/video/VideoStreamManager.java

```diff
@@ -95,6 +97,8 @@
 	private IVideoStreamListener streamListener;
 	private boolean isTransportAvailable = false;
 	private boolean hasStarted;
+	private List<HMILevel> streamableLevels = new ArrayList<>(Arrays.asList(HMILevel.HMI_FULL, HMILevel.HMI_LIMITED));
```

Comment:
Can we simplify it like that? also applies to the other list
```suggestion
	private List<HMILevel> streamableLevels = Arrays.asList(HMILevel.HMI_FULL, HMILevel.HMI_LIMITED);
```